src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.php

```diff
@@ -195,6 +196,12 @@
             ->tag('kernel.event_subscriber')
 
         ->set('messenger.listener.stop_worker_on_stop_exception_listener', StopWorkerOnCustomStopExceptionListener::class)
+
```

Comment:
Is it correct that the `StopWorkerOnCustomStopExceptionListener` service looses the `kernel.event_subscriber` tag?

Reply:
Thanks, fixed in https://github.com/symfony/symfony/pull/43002